packages/publisher/electron-release-server/src/PublisherERS.ts

```diff
@@ -17,6 +17,15 @@ interface ERSVersion {
   assets: { name: string; }[];
 }
 
+type Fetch = typeof fetch;
+const fetchAndCheckStatus = async (...fetchArgs: Parameters<Fetch>): ReturnType<Fetch> => {
+  const result = await fetch(...fetchArgs);
+  if (result.ok) { // res.status >= 200 && res.status < 300
+    return result;
+  }
+  throw new Error(`ERS publish failed with status code: ${result.url}: ${result.statusText}`);
```

Comment:
I think it would make more sense if "status code" and "statusText" were next to each other.

```suggestion
  throw new Error(`ERS publish failed with status code: ${result.statusText} (${result.url})`);
```